src/controllers/processors/InternalEmailFormSubmissionProcessor.ts

```diff
@@ -51,6 +52,10 @@ export class InternalEmailFormSubmissionProcessor implements FormSubmissionProce
             .map(data => data[APPLICATION_DATA_KEY] as ApplicationData)
             .unsafeCoerce();
 
-        await this.emailService.send(buildEmail(userProfile as IUserProfile, applicationData.appeal));
+        const email = buildEmail(userProfile as IUserProfile, applicationData.appeal);
```

Comment:
Same here, `this.emailService.send` performs logging so logging same data here makes no sense to me.

Reply:
removed log in EmailService in favour for the log here